worker/src/lib.cpp

```diff
@@ -126,6 +92,8 @@ extern "C" int mediasoup_worker_run(
 		return 1;
 	}
 
+	// Commenting out the RTC channel setup. Will revisit this if there is a
+	// requirement to set this channel up.
 	/*try
```

Comment:
we should add more details in the comment about the reason behind commenting this out and as part of which effort it was done.